src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,41 +1,71 @@
 package core.basesyntax;
 
+import java.text.DateFormat;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Locale;
+
 public class SalaryInfo {
+
     /**
      * Реализуйте метод getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo)
-     * вычисляющий зарплату сотрудников. На вход методу подаётся 2 массива и 2 даты,
-     * определяющие период за который надо вычислить зарплату, первый массив содержит имена
-     * сотрудников организации, второй массив информацию о рабочих часах и ставке. Формат данных
-     * второго массива следующий: дата, имя сотрудника, количество отработанных часов,
-     * ставка за 1 час. Метод должен вернуть отчёт за период, который передали в метод
-     * (обе даты включительно) составленный по следующей форме: Отчёт за период
-     * #дата_1# - #дата_2# Имя сотрудника - сумма заработанных средств за этот период
-     *
+     * вычисляющий зарплату сотрудников. На вход методу подаётся 2 массива и 2 даты, определяющие
+     * период за который надо вычислить зарплату, первый массив содержит имена сотрудников
+     * организации, второй массив информацию о рабочих часах и ставке. Формат данных второго массива
+     * следующий: дата, имя сотрудника, количество отработанных часов, ставка за 1 час. Метод должен
+     * вернуть отчёт за период, который передали в метод (обе даты включительно) составленный по
+     * следующей форме: Отчёт за период #дата_1# - #дата_2# Имя сотрудника - сумма заработанных
+     * средств за этот период
+     * <p>
      * Пример ввода: date from = 01.04.2019 date to = 30.04.2019
-     *
-     * names:
-     * Сергей
-     * Андрей
-     * София
-     *
-     * data:
-     * 26.04.2019 Сергей 60 50
-     * 26.04.2019 Андрей 3 200
-     * 26.04.2019 Сергей 7 100
-     * 26.04.2019 София 9 100
-     * 26.04.2019 Сергей 11 50
-     * 26.04.2019 Андрей 3 200
-     * 26.04.2019 Сергей 7 100
-     * 26.04.2019 София 9 100
-     * 26.04.2019 Сергей 11 50
-     *
-     * Пример вывода:
-     * Отчёт за период 01.04.2019  - 30.04.2019
-     * Сергей - 1550
-     * Андрей - 600
-     * София - 900
+     * <p>
+     * names: Сергей Андрей София
+     * <p>
+     * data: 26.04.2019 Сергей 60 50 26.04.2019 Андрей 3 200 26.04.2019 Сергей 7 100 26.04.2018
+     * София 9 100 26.04.2019 Сергей 11 50 26.04.2019 Андрей 3 200 26.04.2019 Сергей 7 100
+     * 26.04.2019 София 9 100 26.04.2019 Сергей 11 50
+     * <p>
+     * Пример вывода: Отчёт за период 01.04.2019  - 30.04.2019 Сергей - 1550 Андрей - 600 София -
+     * 900
      */
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        StringBuilder report = new StringBuilder("");
+
```

Comment:
@4MoPDai please take care about code style and empty lines. This is very important when you will write code in production